2023-06-01 16:34:25,436 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Configure stats pid to 21800
2023-06-01 16:34:25,436 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Loading settings from C:\Users\xavi_\.config\wandb\settings
2023-06-01 16:34:25,436 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Loading settings from d:\Math of ML Master - HSE\Skoltech\ICNN\wandb\settings
2023-06-01 16:34:25,436 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2023-06-01 16:34:25,436 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-01 16:34:25,436 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-06-01 16:34:25,440 INFO    MainThread:21800 [wandb_init.py:_log_setup():506] Logging user logs to d:\Math of ML Master - HSE\Skoltech\ICNN\wandb\run-20230601_163425-i88apm2x\logs\debug.log
2023-06-01 16:34:25,440 INFO    MainThread:21800 [wandb_init.py:_log_setup():507] Logging internal logs to d:\Math of ML Master - HSE\Skoltech\ICNN\wandb\run-20230601_163425-i88apm2x\logs\debug-internal.log
2023-06-01 16:34:25,440 INFO    MainThread:21800 [wandb_init.py:init():546] calling init triggers
2023-06-01 16:34:25,440 INFO    MainThread:21800 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'dataset': 'MNIST-FMNIST', 'machine': 'Colab', 'model': 'resnet_d', 'loss_function': 'l1_loss', 'image_size': 32, 'learning_rate': 0.001, 'batch_size_loop': 64, 'batch_size_paired': 128, 'epochs': 3000}
2023-06-01 16:34:25,440 INFO    MainThread:21800 [wandb_init.py:init():568] re-initializing run, found existing run on stack: 4627ke48
2023-06-01 16:34:25,448 INFO    MainThread:21800 [wandb_run.py:_finish():1854] finishing run icnn-23/FMNIST-resnet_d-False-l1_loss-t/4627ke48
2023-06-01 16:34:25,452 ERROR   MainThread:21800 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2023-06-01 16:34:25,452 INFO    MainThread:21800 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-06-01 16:34:25,456 INFO    MainThread:21800 [wandb_init.py:_jupyter_teardown():434] cleaning up jupyter logic
2023-06-01 16:34:25,456 INFO    MainThread:21800 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-06-01 16:34:25,456 INFO    MainThread:21800 [wandb_run.py:_restore():2066] restore
2023-06-01 16:34:25,456 INFO    MainThread:21800 [wandb_run.py:_restore():2072] restore done
2023-06-01 16:34:31,584 INFO    MainThread:21800 [wandb_run.py:_footer_history_summary_info():3422] rendering history
2023-06-01 16:34:31,584 INFO    MainThread:21800 [wandb_run.py:_footer_history_summary_info():3454] rendering summary
2023-06-01 16:34:31,597 INFO    MainThread:21800 [wandb_run.py:_footer_sync_info():3380] logging synced files
2023-06-01 16:34:31,612 INFO    MainThread:21800 [wandb_init.py:init():602] starting backend
2023-06-01 16:34:31,612 INFO    MainThread:21800 [wandb_init.py:init():606] setting up manager
2023-06-01 16:34:31,633 INFO    MainThread:21800 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-06-01 16:34:31,653 INFO    MainThread:21800 [wandb_init.py:init():613] backend started and connected
2023-06-01 16:34:31,672 INFO    MainThread:21800 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2023-06-01 16:34:31,672 INFO    MainThread:21800 [wandb_run.py:_label_probe_notebook():1214] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-06-01 16:34:31,672 INFO    MainThread:21800 [wandb_init.py:init():701] updated telemetry
2023-06-01 16:34:31,792 INFO    MainThread:21800 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-06-01 16:34:33,266 INFO    MainThread:21800 [wandb_run.py:_on_init():2133] communicating current version
2023-06-01 16:34:33,530 INFO    MainThread:21800 [wandb_run.py:_on_init():2142] got version response upgrade_message: "wandb version 0.15.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-06-01 16:34:33,540 INFO    MainThread:21800 [wandb_init.py:init():789] starting run threads in backend
2023-06-01 16:34:33,958 INFO    MainThread:21800 [wandb_run.py:_console_start():2114] atexit reg
2023-06-01 16:34:33,958 INFO    MainThread:21800 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-06-01 16:34:33,958 INFO    MainThread:21800 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-06-01 16:34:33,960 INFO    MainThread:21800 [wandb_run.py:_redirect():2059] Redirects installed.
2023-06-01 16:34:33,960 INFO    MainThread:21800 [wandb_init.py:init():831] run started, returning control to user process
2023-06-01 16:35:40,742 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Configure stats pid to 21800
2023-06-01 16:35:40,742 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Loading settings from C:\Users\xavi_\.config\wandb\settings
2023-06-01 16:35:40,748 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Loading settings from d:\Math of ML Master - HSE\Skoltech\ICNN\wandb\settings
2023-06-01 16:35:40,748 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2023-06-01 16:35:40,750 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-01 16:35:40,750 INFO    MainThread:21800 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-06-01 16:35:40,750 INFO    MainThread:21800 [wandb_init.py:_log_setup():506] Logging user logs to d:\Math of ML Master - HSE\Skoltech\ICNN\wandb\run-20230601_163540-elyqawym\logs\debug.log
2023-06-01 16:35:40,750 INFO    MainThread:21800 [wandb_init.py:_log_setup():507] Logging internal logs to d:\Math of ML Master - HSE\Skoltech\ICNN\wandb\run-20230601_163540-elyqawym\logs\debug-internal.log
2023-06-01 16:35:40,750 INFO    MainThread:21800 [wandb_init.py:_jupyter_setup():452] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002668100B4F0>
2023-06-01 16:35:40,750 INFO    MainThread:21800 [wandb_init.py:init():546] calling init triggers
2023-06-01 16:35:40,750 INFO    MainThread:21800 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {'dataset': 'MNIST-FMNIST', 'machine': 'Colab', 'model': 'resnet_d', 'loss_function': 'l1_loss', 'image_size': 32, 'learning_rate': 0.001, 'batch_size_loop': 64, 'batch_size_paired': 128, 'epochs': 3000}
2023-06-01 16:35:40,750 INFO    MainThread:21800 [wandb_init.py:init():568] re-initializing run, found existing run on stack: i88apm2x
2023-06-01 16:35:40,760 INFO    MainThread:21800 [wandb_run.py:_finish():1854] finishing run icnn-23/FMNIST-resnet_d-False-l1_loss-t/i88apm2x
2023-06-01 16:35:40,760 INFO    MainThread:21800 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-06-01 16:35:40,760 INFO    MainThread:21800 [wandb_run.py:_restore():2066] restore
2023-06-01 16:35:40,760 INFO    MainThread:21800 [wandb_run.py:_restore():2072] restore done
2023-06-01 16:35:45,351 INFO    MainThread:21800 [wandb_run.py:_footer_history_summary_info():3422] rendering history
2023-06-01 16:35:45,351 INFO    MainThread:21800 [wandb_run.py:_footer_history_summary_info():3454] rendering summary
2023-06-01 16:35:45,376 INFO    MainThread:21800 [wandb_run.py:_footer_sync_info():3380] logging synced files
